CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
   - Emit entity events only for matching receivers ([#4691](https://github.com/signalfx/splunk-otel-collector/pull/4691))
   - Remove `log_endpoints` config option ([#4692](https://github.com/signalfx/splunk-otel-collector/pull/4692))
 
+### 🚩 Deprecations 🚩
+
+- (Splunk) `collectd-mongodb`: Monitor has been removed to resolve CVE-2024-21506 ([#4731](https://github.com/signalfx/splunk-otel-collector/pull/4731))
```

Comment:
Should this be a breaking change instead?

Reply:
Sure. I had it under breaking change at first but thought `deprecation` might be slightly more accurate. I'll move it to breaking 👍